dashboard/app/models/user.rb

```diff
@@ -2185,6 +2185,12 @@ def record_soft_delete
     )
   end
 
+  def donor_school
+    if school_info
+      DonorSchool.find_by(nces_id: school_info.id)
```

Comment:
Pretty sure this needs to be
```rb
if last_complete_school_info&.school
  DonorSchool.find_by(nces_id: last_complete_school_info.school.id)
```

Reply:
thank you! I was pretty sure this was not right!